odkx-src/xlsx-converter-reference.rst

```diff
@@ -391,6 +391,19 @@ The following prompt types are available in ODK-X Survey.
       | like to enter.
   * - video
     - | Used to capture a video.
+  * - textarea
+    - | Used to enter the information in a big text area or paragraphs.
+    
+.. note::
+   if users anticipates for writing anything longer than 255 characters then the user needs to change the model sheet and change the elementType column. It is shown in the datatypes XLSX, string variables' length can be adjusted from a default of 255 to other lengths with string(len). For example, if you had a string prompt named long_data that you wanted to be 500 characters, you would add the following to your model worksheet. To know more refer to https://docs.odk-x.org/xlsx-converter-reference/#model
```

Comment:
Please use a `ref` to link to that section. https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations